packages/contracts-ops/contracts/scripts/DeployAccountant.s.sol

```diff
@@ -41,14 +40,30 @@ abstract contract DeployAccountantLogic is Script, Config {
             getAccountantOwner(_domain)
         );
     }
+
+    function updateAccountant(string memory _domain, string memory _configPath)
```

Comment:
function does not update the accountant, it writes an existing update to the accountant json. suggested name: `writeAccountantImplementation` or similar

Reply:
okay, can change to `write` but the whole upgrade setup (not just implementation) needs to be written

Reply:
I renamed to write, but did not change behavior

Reply:
https://github.com/nomad-xyz/monorepo/pull/511/commits/b758804fea4adf32e72569a32d4d3683320de04b